index.js

```diff
@@ -207,11 +207,15 @@ class ExecutorQueue extends Executor {
         }
 
         return new Promise((resolve, reject) => {
-            req(options, (err, response) => {
+            req(options, (err, response, body) => {
                 if (!err && response.statusCode === 201) {
                     return resolve(response);
                 }
 
+                if (response.statusCode !== 201) {
```

Comment:
if we enter this condition, the following `reject` never gets executed/

Reply:
why?